recipe/meta.yaml

```diff
@@ -32,14 +32,14 @@ requirements:
     - pandas
     - astropy
     - matplotlib
-    - qtpy
+    - qt 4.8.*
```

Comment:
@astrofrog the bad replacements is clearly a bug in the pinning script. Also, the script does not understand the concept of `!=`. So do not worry about this change here until we fix that.

Ping @pelson.